css/style.css

```diff
@@ -884,6 +1013,52 @@ html {
     overflow-y: scroll;
   }
 
+  .pop-up-container {
+    display: flex;
+    flex-direction: column;
+    width: 48rem;
+    max-height: 65.1rem;
+  }
```

Comment:
- [OPTIONAL] Kindly make sure the pop-up container matches the provided Figma design as much as possible on the mobile screens.

  - ### YOUR DESIGN
  
      ![image](https://user-images.githubusercontent.com/87197412/216328638-562cb917-0577-44aa-9d5f-bd17503ce9dc.png)

  - ### FIGMA DESIGN
  
     ![image](https://user-images.githubusercontent.com/87197412/216328739-08bdbc79-d4cb-4301-a1be-c0bbfc238c33.png)